recipes/hdf5/all/conanfile.py

```diff
@@ -59,13 +62,13 @@ def validate(self):
         if not can_run(self):
             # While building it runs some executables like H5detect
             raise ConanInvalidConfiguration("Current recipe doesn't support cross-building (yet)")
-        if self.info.options.parallel:
-            if self.info.options.enable_cxx:
+        if self.options.parallel and not self.options.enable_unsupported:
+            if self.options.enable_cxx:
                 raise ConanInvalidConfiguration("Parallel and C++ options are mutually exclusive")
```

Comment:
Should the comment be updated to include the new option?

Reply:
Yes, I think it should. Done.